corehq/apps/export/forms.py

```diff
@@ -407,17 +407,13 @@ def layout_fields(self):
             )
         ]
 
-    def to_export_instance_filters(self, can_access_all_locations, accessible_location_ids):
+    def to_export_instance_filters(self, can_access_all_locations, accessible_location_ids, export_type):
         """
         Serialize the bound form as an ExportInstanceFilters object.
         """
-        # Confirm that either form filter data or case filter data but not both has been submitted.
-        assert (
```

Comment:
This assert still seems useful, since it's testing something different than the assert you added (which also seems useful).